### PR TITLE
feat: space id name service

### PR DESCRIPTION
**Pull request by jincan39**

### CODE REVIEW

File: docs/manta-pacific/Space ID.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+There are optional parameters in the method to select your target chain or TLD (top-level Domain).
+
+By providing chain IDs, you can resolve addresses on selected chains and get an available domain name from all TLDs deployed on these chains.
+
+```javascript
+// Resolve an address from Gnosis Chiado
+const name = await web3name.getDomainName({
+    address: "0x2886D6792503e04b19640C1f1430d23219AF177F",
+    queryChainIdList: [10200],
```
Comment (eddddiez) [inline docs/manta-pacific/Space ID.md:91]: change id to mantan's mainnet id

File: docs/manta-pacific/Space ID.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+// expect: lydia.gno
+```
+
+By providing TLDs, address can be resolved from the selected TLDs and get an available TLD primary name.
+
+```javascript
+// Resolve an address from .gno TLD
+const name = await web3name.getDomainName({
+    address: "0x2886D6792503e04b19640C1f1430d23219AF177F",
+    queryTldList: ["gno"],
```
Comment (eddddiez) [inline docs/manta-pacific/Space ID.md:102]: can change to manta

File: docs/manta-pacific/Space ID.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+We are using popular public RPC services by default to make it easier to use. But in some cases developers may prefer to use arbitrary RPC, so we provide optional parameter `rpcUrl` for each function that allows developers to use their own RPC to make requests.
+For example, you can put custom rpcUrl as a parameter in `getAddress` function.
+
+For example, you can put custom rpcUrl as a parameter in getAddress function.
+
+```javascript
+// Use custom RPC url (https://arb1.arbitrum.io/rpc)
+const address = await web3name.getAddress("registry.arb", {
+    rpcUrl: "https://arb1.arbitrum.io/rpc",
```
Comment (eddddiez) [inline docs/manta-pacific/Space ID.md:137]: use manta rpc as well

File: docs/manta-pacific/Space ID.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+For other functions, it's also possible to have a custom `rpcUrl` in the request.
+
+```javascript
+// Use custom RPC url (https://arb1.arbitrum.io/rpc)
+const address = await web3name.getMetaData("registry.arb", {
+    rpcUrl: "https://arb1.arbitrum.io/rpc",
+});
+// expect: '0x8d27d6235d9d8EFc9Eef0505e745dB67D5cD2918'
+```
```
Comment (eddddiez) [inline docs/manta-pacific/Space ID.md:150]: remove all the expected fields

File: sidebars.js [hunk 5]
```diff
@@ -182,6 +182,11 @@ module.exports = {
                     label: "RedStone Oracles",
                     id: "manta-pacific/RedStone Oracles",
                 },
+                {
+                    type: "doc",
+                    label: "Space ID Name Service",
+                    id: "manta-pacific/Space ID",
```
Comment (eddddiez) [inline sidebars.js:188]: SPACE ID Name Service and SPACE ID. Capitalize space.

**Review by jasonintju — APPROVED** [review]

**Pull request merged**